GVFS/GVFS.Service/Handlers/EnableAndAttachProjFSHandler.cs

```diff
@@ -87,9 +87,26 @@ public static bool TryEnablePrjFlt(ITracer tracer, out string error)
                 isNativeLibInstalled = ProjFSFilter.IsNativeLibInstalled(tracer, new PhysicalFileSystem());
                 if (!isNativeLibInstalled)
                 {
-                    string missingNativeLibMessage = "Native library is not installed";
-                    error = string.IsNullOrEmpty(error) ? missingNativeLibMessage : $"{error}. {missingNativeLibMessage}";
-                    tracer.RelatedError($"{nameof(TryEnablePrjFlt)}: {missingNativeLibMessage}");
+                    if (isRunning)
+                    {
+                        tracer.RelatedInfo($"{nameof(TryEnablePrjFlt)}: Native ProjFS library is not installed, attempting to copy packaged version");
+                        string copyNativeDllError;
+                        if (ProjFSFilter.TryCopyNativeLibIfDriverVersionsMatch(tracer, new PhysicalFileSystem(), out copyNativeDllError))
+                        {
+                            tracer.RelatedInfo($"{nameof(TryEnablePrjFlt)}: Successfully copied packaged ProjFS native library");
+                            isNativeLibInstalled = true;
+                        }
+                        else
+                        {
+                            tracer.RelatedError($"{nameof(TryEnablePrjFlt)}: Failed to copy packaged ProjFS native library {copyNativeDllError}");
```

Comment:
Do we need to set the error message in this code path? It seems like before we set an error if we got to this point, but were missing the native ProjFS library.